addons/skin.estuary/xml/Variables.xml

```diff
@@ -122,9 +122,14 @@
 		<value condition="String.IsEqual(listitem.dbtype,musicvideo) | String.IsEqual(listitem.dbtype,video)">$INFO[ListItem.Genre,[COLOR button_focus]$LOCALIZE[515]: [/COLOR],[CR]]$INFO[ListItem.Plot]</value>
 		<value condition="String.IsEqual(listitem.dbtype,artist)">$INFO[ListItem.Property(artist_description)]</value>
 		<value condition="!String.IsEmpty(ListItem.Plot)">$INFO[ListItem.Plot]</value>
-		<value condition="String.IsEqual(listitem.dbtype,song)">$INFO[listitem.TrackNumber,[COLOR button_focus]$LOCALIZE[554]: [/COLOR]]. $INFO[ListItem.Title,,[CR]]$INFO[ListItem.Artist,[COLOR button_focus]$LOCALIZE[557]: [/COLOR],[CR]]$INFO[listitem.Album,[COLOR button_focus]$LOCALIZE[558]: [/COLOR],[CR]]$INFO[ListItem.DiscNumber,[COLOR button_focus]$LOCALIZE[427]: [/COLOR],[CR]]$INFO[ListItem.Year,[COLOR button_focus]$LOCALIZE[345]: [/COLOR],[CR]]$INFO[ListItem.Genre,[COLOR button_focus]$LOCALIZE[515]: [/COLOR],[CR]]$INFO[ListItem.Duration,[COLOR button_focus]$LOCALIZE[180]: [/COLOR],[CR]]$INFO[ListItem.Playcount,[COLOR button_focus]$LOCALIZE[567]: [/COLOR],[CR]]</value>
+		<value condition="String.IsEqual(listitem.dbtype,song)">$VAR[MusicTrackInfo,[COLOR button_focus]$LOCALIZE[554]:[/COLOR],[CR]]$INFO[ListItem.Artist,[COLOR button_focus]$LOCALIZE[557]: [/COLOR],[CR]]$INFO[listitem.Album,[COLOR button_focus]$LOCALIZE[558]: [/COLOR],[CR]]$INFO[ListItem.DiscNumber,[COLOR button_focus]$LOCALIZE[427]: [/COLOR],[CR]]$INFO[ListItem.Year,[COLOR button_focus]$LOCALIZE[345]: [/COLOR],[CR]]$INFO[ListItem.Genre,[COLOR button_focus]$LOCALIZE[515]: [/COLOR],[CR]]$INFO[ListItem.Duration,[COLOR button_focus]$LOCALIZE[180]: [/COLOR],[CR]]$INFO[ListItem.Playcount,[COLOR button_focus]$LOCALIZE[567]: [/COLOR],[CR]]</value>
 		<value>$INFO[ListItem.Genre,[COLOR button_focus]$LOCALIZE[515]: [/COLOR],[CR]]</value>
 	</variable>
+	<variable name="MusicTrackInfo">
+		<value condition="!String.IsEmpty(listitem.TrackNumber) + !String.IsEmpty(listitem.Title">$INFO[listitem.TrackNumber, ,.]$INFO[listitem.Title, ]</value>
+		<value condition="!String.IsEmpty(listitem.TrackNumber) + String.IsEmpty(listitem.Title">$INFO[listitem.TrackNumber, ]</value>
+		<value condition="String.IsEmpty(listitem.TrackNumber) + !String.IsEmpty(listitem.Title">$INFO[listitem.Title, ]</value>
+	</variable>	
```

Comment:
i think this can be simplified to:
```	
<variable name="MusicTrackInfo">
	<value condition="String.IsEmpty(listitem.Title)">$INFO[listitem.TrackNumber, ]</value>
	<value>$INFO[listitem.TrackNumber, ,.]$INFO[listitem.Title, ]</value>
</variable>
```

Reply:
Looks like that would also work, but will need to do a thorough check later.